docker/sasl/docker-compose.yml

```diff
@@ -43,3 +43,33 @@ services:
       KAFKA_OPTS: '-Djava.security.auth.login.config=/etc/kafka/kafka_server_jaas.conf'
       KAFKA_LOG4J_LOGGERS: "kafka.authorizer.logger=DEBUG"
 
+  control-center:
```

Comment:
Why C3 is required here?

Reply:
because the way I use this dockers is for testing and having C3 allows me to properly test for custom internal topics ;-), automation is as well introduced in the test to test and verify for custom internal topics.